roles/stats_agent/templates/push_metrics.sh.j2

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set variables
+RPC_URL="http://localhost:{{ zilliqa.validator.ports.api_port }}/"
+PUSHGATEWAY_URL="http://{{ monitor_server_dns }}:9091/metrics/job/zilliqa_metrics"
```

Comment:
does it have to be a new job name ? If no, lets use chain_exporter https://github.com/encapsulate-xyz/monitor-ansible/blob/main/roles/prometheus/templates/prometheus_config.yml.j2#L76C16-L76C30